index.js

```diff
@@ -62,7 +62,7 @@ window.onload = () => {
         !isLocalDev && window.DD_RUM && window.DD_RUM.init({ ...datadogConfig, env: 'dev' });
     }
 
-    !isLocalDev && window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
+    !isLocalDev && !(location.host === urls.prod) && window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
```

Comment:
```suggestion
    !isLocalDev && location.host !== urls.prod && window.DD_RUM && window.DD_RUM.startSessionReplayRecording();
```

Reply:
should it be `!=` or `!==` ? Not sure if there is an actual difference